src/hooks/useAccommodationsAndOffers.tsx/index.ts

```diff
@@ -27,7 +27,13 @@ export const useAccommodationsAndOffers = (props: SearchTypeProps | void) => {
       }
       return await fetchAccommodationsAndOffers(props);
     },
-    { enabled: false, keepPreviousData: false }
+    {
+      enabled: false,
```

Comment:
Don't we have to show UI displaying that the offers expire?

Reply:
how?why?I thought its better to be hidden